setup.py

```diff
@@ -57,10 +58,7 @@ def add_files(level=[]):
     data_files = [(get_dir(['etc', 'avocado', 'conf.d']),
                    ['etc/avocado/conf.d/vt.conf'])]
 
-    data_files += [(get_dir(['usr', 'share', 'avocado-plugins-vt',
-                             'test-providers.d']),
-                    glob.glob('test-providers.d/*'))]
-
+    data_files += add_files(["test-providers.d"])
```

Comment:
Nop, the `data_files_dirs` handling adds 2nd+ level dirs only and `test-providers.d` contains directly the files we need to add.